infrastructure/src/test/java/org/corfudb/infrastructure/SequencerServerTest.java

```diff
@@ -470,17 +468,15 @@ public void testStreamAddressRequest() {
                         getStreamAddressSpace(uuidToStreamAddressSpacePairMsg.getAddressSpace());
                 assertEquals(globalTailB,
                         finalResponse.getPayload().getStreamsAddressResponse().getLogTail());
-                assertEquals(Roaring64NavigableMap.bitmapOf(globalTailA),
-                        responseStreamAddressSpaceA.getAddressMap());
+                assertEquals(new StreamAddressSpace(Collections.singleton(globalTailA)), responseStreamAddressSpaceA);
                 assertEquals(Address.NON_ADDRESS, responseStreamAddressSpaceA.getTrimMark());
             } else {
                 StreamAddressSpace responseStreamAddressSpaceB =
                         getStreamAddressSpace(uuidToStreamAddressSpacePairMsg.getAddressSpace());
                 assertEquals(globalTailB,
                         finalResponse.getPayload().getStreamsAddressResponse().getLogTail());
-                assertEquals(Roaring64NavigableMap.bitmapOf(globalTailB),
-                        responseStreamAddressSpaceB.getAddressMap());
-                assertEquals(Address.ABORTED, responseStreamAddressSpaceB.getTrimMark());
+                assertEquals(new StreamAddressSpace(Collections.singleton(globalTailB)), responseStreamAddressSpaceB);
+                assertEquals(Address.NON_ADDRESS, responseStreamAddressSpaceB.getTrimMark());
```

Comment:
same here, 479 can be removed

Reply:
We have to check that both StreamAddressSpace objects are the same AND that the trim mark is NON_ADDRESS.